style.css

```diff
@@ -0,0 +1,75 @@
+#container{
```

Comment:
Of course it’s not our subject, but it is not suitable to use id in css. Use styles and css selectors etc.